src/main/java/nl/dtls/fairdatapoint/database/mongo/migration/production/Migration_0006_ShapesSharing.java

```diff
@@ -1,17 +1,17 @@
 /**
  * The MIT License
  * Copyright © 2017 DTL
- * <p>
+ *
  * Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
The other newly added license header (`Migration_0007_RemoveMongobee.java`) seems to have the `<p>`. Needs to be removed there too? `mvn license:format` should be able to handle that.

Reply:
`mvn license:format` does this: https://github.com/FAIRDataTeam/FAIRDataPoint/pull/129/commits/0d0f46bb3a0855f2781740533093933eccbcc0f4

Do we want to change all the files now? If so, should we do also another change in the license?

Reply:
Fix by adding this to our pom:

```xml
<mapping>
    <java>JAVADOC_STYLE</java>
</mapping>
```

It now default Java to `SLASHSTAR_STYLE`. Also IDE marks `/**` as "dangling javadoc", but lets not mess with the git history at the moment 😄 More here: https://github.com/mycila/license-maven-plugin#supported-comment-types